### ISSUE TITLE
How to perform classification instead of default regression?

### ISSUE THREAD

**Issue by pjspol** [gmum/huggingmolecules#33]
The quick tour shows only regression, and when I substitute in a classification loss function and metric, the model is still behaving like a regressor. Is there something with which to specify a 'classification' task?

Thank you!

**Comment by panpiort8** [comment]
It's because for binary classification model outputs a single value (logit), which should be lower for 0 class and higher for 1. Logit can be then feed to sigmoid to obtain value in [0, 1] that can be roughly seen as (uncalibrated) probability of class 1. 

So, basically, regression and binary classification models are distinguishable only in terms of loss function and data provided for training,
